hardware/raspberrypi/power/README.md

```diff
@@ -15,8 +15,8 @@ Exactly how much current (mA) the Raspberry Pi requires is dependent on what you
 | Raspberry Pi 3 Model A+ | 2.5A | Limited by PSU, board, and connector ratings only. | 350mA |
 | Raspberry Pi 3 Model B+ | 2.5A | 1.2A | 500mA |
 | Raspberry Pi 4 Model B | 3.0A | 1.2A | 600mA |
-| Raspberry Pi Zero W/WH | 1.2A | Limited by PSU, board, and connector ratings only.| 150mA |
 | Raspberry Pi Zero | 1.2A | Limited by PSU, board, and connector ratings only | 100mA |
```

Comment:
Seems weird that the Zero has a higher "Recommended PSU current capacity" than the A+, but meh :man_shrugging:

Reply:
It's clocked higher IIRC, so that will require a bit more ooomph. Did you seem comment re: under voltage warnings?